vnext/ReactUWP/Modules/TimingModule.cpp

```diff
@@ -144,12 +144,20 @@ void Timing::createTimer(
 
   if (m_timerQueue.IsEmpty()) {
```

Comment:
Will this be a problem when background task is running and the app is bring to front?

Reply:
That's a good point.
With this implementation the Timer will fire once the App moves to foreground and the Rendering event is hooked up.

Intuitively I think it is a little strange.

But I went with this because it matches the Android behavior for react native.
Specifically this quote

> The function passed to setTimeout does not always behave as expected. Instead the function is called only when the application is launched again. If you just need to wait, use the retry functionality.

https://facebook.github.io/react-native/docs/0.60/headless-js-android#caveats

Reply:
Can you add comment on the code?

Reply:
Updated